src/main/java/ee/tuleva/onboarding/fund/statistics/PensionFundStatisticsService.java

```diff
@@ -60,12 +60,12 @@ public PensionFundStatisticsService(RestTemplateBuilder restTemplateBuilder) {
 
             if (result == null) {
                 log.info("Pension fund statistics is empty");
-                return emptyList();
+                return new ArrayList<>();
```

Comment:
wai

Reply:
emtpyList() is immutable so -> `UnsupportedOperationException
add`